core/src/epicli/data/common/ansible/playbooks/roles/postgresql/molecule/README.adoc

```diff
@@ -0,0 +1,35 @@
+= Installation guide
```

Comment:
Why not MD?:) Also would be cool if we can run these tests from the container and not only externally.

Reply:
This is a personal preference, I will change that to markdown.